engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -869,3 +869,6 @@ WHERE   usage_unit = 'Policy-Month';
 
 -- delete configuration task.cleanup.retry.interval #6910
 DELETE FROM `cloud`.`configuration` WHERE name='task.cleanup.retry.interval';
+
+--- #6888 add index to speed up querying IPs in the network-tab
+ALTER TABLE `user_ip_address` ADD KEY `fk_user_ip_address__state` (`state`);
```

Comment:
@DaanHoogland 
`state` is not a foreign key.
It would be good to use `i_user_ip_address__state`

might be better to create a procedure similar as IDEMPOTENT_ADD_FOREIGN_KEY ?

Reply:
The procedure should already exist, so calling like
```suggestion
CALL `cloud`.`IDEMPOTENT_CREATE_UNIQUE_INDEX`('uk_user_ip_address_state','user_ip_address', '(state)');
```